server/cpho/views/indicator_metadata.py

```diff
@@ -506,6 +506,26 @@ def clean_trend_segment(self):
             trend_segment = trend_segment.strip().replace(" ", "")
             return trend_segment
 
+    def clean_data_point_lower_ci(self):
+        data_point = self.cleaned_data["data_point"]
+        data_point_lower_ci = self.cleaned_data["data_point_lower_ci"]
+        if data_point_lower_ci and data_point_lower_ci > data_point:
+            self.add_error(
+                "data_point_lower_ci",
+                tm("data_point_lower_ci_error"),
+            )
+        return data_point_lower_ci
+
+    def clean_data_point_upper_ci(self):
+        data_point = self.cleaned_data["data_point"]
+        data_point_upper_ci = self.cleaned_data["data_point_upper_ci"]
+        if data_point_upper_ci and data_point_upper_ci < data_point:
+            self.add_error(
+                "data_point_upper_ci",
+                tm("data_point_upper_ci_error"),
+            )
+        return data_point_upper_ci
```

Comment:
the `clean_<field>()` methods should typically only look at their own data, because other data is not guaranteed to exist. For instance, now we get 500s from line 510 is data-point is left empty

We can move these 2 checks into clean() or we can add extra checks to make sure data_point is not none